libstuff/SPerformanceTimer.cpp

```diff
@@ -6,8 +6,9 @@ SPerformanceTimer::SPerformanceTimer(string description, bool reverse, uint64_t
       _lastLogStart(0), _timeLogged(0), _timeNotLogged(0), _description(description) {}
 
 SPerformanceTimer::~SPerformanceTimer() {
-    // The final shutdown.
-    log();
+    // TODO: We'd like to do one final log at shutdown, but this breaks for statically allocated timers, because we
+    // won't have SINFO available when this destructor is called. Thus, this is currently disabled.
```

Comment:
Is this ever coming back?  It feels like this is going to stay disabled forever.  Can we just remove this?